contracts/Hasher.sol

```diff
@@ -93,4 +93,16 @@ contract Hasher {
 			_unlockHeight,
 			_hashLock);
 	}
+
+	function hashIntentKey(
+		address _staker,
```

Comment:
More generic name for Staker ?
Planning to use this contract for OpenSTUtility too to store the redemptionIntentHash.

Reply:
Are you requesting me to change this to a more generic term and replace `_staker`? Would you have any suggestions? How about simply using `_account` ?

Reply:
Yes !!! sorry for not being clear while commenting.  ``` _account ``` sounds good to me.